lib/streamlit/config.py

```diff
@@ -296,6 +293,56 @@ def _global_unit_test():
     type_=int,
 )
 
+
+# Config Section: Logger #
+_create_section("logger", "Settings to customize Streamlit log messages.")
+
+
+@_create_option("logger.datetimeFormat", type_=str)
+def _logger_datetime_format():
+    """Datetime format for date in logger message. Providing a datetime format
+    will automatically add a timestamp to all logger messages.
+
+    Default: None
+    """
+    if get_option("global.developmentMode"):
```

Comment:
I think we should have timestamps with _all_ logs, and then provide a way to turn it _off_ if desired.

Reply:
I would be fine with that but S4A would then have to turn it off by default or at least re-evaluate time-stamping logs. See [S4A discussion](https://github.com/streamlit/s4t/pull/227)